src/indent-alignment.js

```diff
@@ -75,6 +75,27 @@ module.exports = {
          );
       });
 
+      // Ensure top-level lists adhere to the start_indent setting if it`s defined
+      if (params.config.start_indent !== undefined) {
+         iterate(params.parsers.micromark.tokens, [ 'listOrdered', 'listUnordered' ], (token) => {
+            const startColumn = token.startColumn - 1;
+
+            addErrorDetailIf(
+               onError,
+               token.startLine,
+               params.config.start_indent,
```

Comment:
After testing I noticed I inverted the 'expected' and 'actual' params. This should be fixed now.